src/main/java/gg/solarmc/kitpvp/messaging/Prefix.java

```diff
@@ -1,23 +1,54 @@
 package gg.solarmc.kitpvp.messaging;
 
+import org.bukkit.Server;
 import org.bukkit.entity.Player;
 
+import java.awt.*;
```

Comment:
I don't recommend wildcard imports in general, and especially not for java.awt, because of java.awt.List whose presence may clash with an intention of using a java.util.List

Reply:
I don't believe i added that myself, rather the IntelliJ Auto import feature must have added it as i don't manage the imports myself